Source/Core/DolphinQt/Debugger/BranchWatchDialog.h

```diff
@@ -111,9 +119,16 @@ class BranchWatchDialog : public QDialog
   void Load(const Core::CPUThreadGuard& guard, const std::string& filepath);
   void AutoSave(const Core::CPUThreadGuard& guard);
   void SetStubPatches(u32 value) const;
+  void SetEditPatches(u32 (*transform)(u32)) const;
   void SetBreakpoints(bool break_on_hit, bool log_on_hit) const;
 
-  [[nodiscard]] QMenu* GetTableContextMenu(const QModelIndex& index);
+  void SetBreakpointMenuActionsIcons();
+  QMenu* GetTableContextMenu(const QModelIndex& index);
+  QMenu* GetTableContextMenu_Instruction(bool core_initialized);
+  QMenu* GetTableContextMenu_Condition(bool core_initialized);
+  QMenu* GetTableContextMenu_Origin(bool core_initialized);
+  QMenu* GetTableContextMenu_Destin(bool core_initialized);
+  QMenu* GetTableContextMenu_Symbol(bool core_initialized);
```

Comment:
Can't these be `const` methods? AFAICT, `this`'s members aren't mutated. It's just calling methods from them.

Reply:
Ah, that method used to mutate a pointer in the `BranchWatchDialog`, but after my refactor it no longer does. Good catch.